src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall () {
```

Comment:
getRandomBall() should return Ball, not string
```suggestion
    public Ball getRandomBall() {
```